requirements.txt

```diff
@@ -1 +1,2 @@
-palmcli >=2.0, <3.0
\ No newline at end of file
+palmcli >=2.0, <3.0
```

Comment:
We will need to adjust the version pinning here, since we need the Containerizer stuff in palm-cli. This is a new feature, so palmcli will go to v2.1 when this code is released. I'll bump the version in the corresponding palm-cli branch now so that this code will continue to work when you're running it locally